frontend/src/layout/header/UserBox.tsx

```diff
@@ -419,11 +438,16 @@ const MenuItem: React.FC<MenuItemProps> = ({
         }
     };
 
+    const additionalProps = !isLanguageItem ? { role: "menuitem" } : {
+        role: "checkbox",
+        "aria-checked": selected,
+    };
+
     return linkTo
         ? <li role="menuitem" {... { className }}>
```

Comment:
Should `additionalProps` be used here as well? I mean, this path is currently not used, but seems weird to only use it in one case.